go.mod

```diff
@@ -108,5 +108,3 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sp
 replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor => ./processor/resourcedetectionprocessor/
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor => ./processor/metricstransformprocessor/
-
-replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
```

Comment:
This was one of the hardest to find problem :(

Reply:
Glad this hack is finally not needed!

Reply:
Unfortunately I had to add a different one, but that should be easier to remove, only need to update k8s.io/client-go > 0.19.x :)